.changeset/selfish-ghosts-live.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-neoboard-widget': minor
+---
+
+It is now possible to change the colours of multiple selected elements
```

Comment:
This would be imperative mood then and wouldn't make much sense to me if I read it. I would suggest to take the changelog format to discussion. Maybe it should even be defined by product people as part of the issue.

Applying the change now to get this PR forward :fast_forward: